web-local/src/lib/components/entity-mappers/mappers.ts

```diff
@@ -0,0 +1,47 @@
+import { EntityType } from "@rilldata/web-local/common/data-modeler-state-service/entity-state-service/EntityStateService";
```

Comment:
This file `entity-mappers/mappers.ts` should be in the `utils/` folder not the `components/` folder, right?